### DISCUSSION TITLE
Where is OutputPath folder for Uno projects?

### DISCUSSION THREAD

**Discussion by sam-wheat** [unoplatform/uno#10639] [Q&A] [answered]
I have about a dozen projects that contain this directive in the .csproj:

	<Target Name="CopyPackage" AfterTargets="Pack" Condition=" '$(Configuration)'=='Debug' ">
		<Copy SourceFiles="$(OutputPath)..\$(PackageId).$(PackageVersion).nupkg" DestinationFolder="c:\NugetPackages\" />
		<Copy SourceFiles="$(OutputPath)..\$(PackageId).$(PackageVersion).snupkg" DestinationFolder="c:\NugetPackages\" />
	</Target>	
	
	
All of these project output a .nupkg file in bin/Debug and copy it as specified.

My uno project does not work - it generates the .nupkg in bin/Debug as other projects do however I get this build error:

> Error	MSB3030	Could not copy the file "bin\Debug\..\MyProject.1.0.0-beta.1.nupkg" because it was not found.	

When I change the directive as follows it works fine (remove .. after OutputPath):

	  <Target Name="CopyPackage" AfterTargets="Pack" Condition=" '$(Configuration)'=='Debug' ">
		<Copy SourceFiles="$(OutputPath)\$(PackageId).$(PackageVersion).nupkg" DestinationFolder="c:\NugetPackages\" />
		<Copy SourceFiles="$(OutputPath)\$(PackageId).$(PackageVersion).snupkg" DestinationFolder="c:\NugetPackages\" />
	  </Target>
	  
The weierd thing about is that all my projects - including the Uno project - output the .nupkg to bin/Debug.

**Comment by jeromelaban** [comment, marked as answer]
MSBuild NuGet targets provide an existing property called [`PackageOutputPath`](https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target-inputs) that you can use to avoid copying packages after the build.
